client/components/repository/Workflow/Sidebar/index.vue

```diff
@@ -13,18 +13,23 @@
       <status-field-group
         @update="updateStatus"
         v-bind="selectedActivity.status"
-        class="mt-9 mb-12" />
+        class="mt-9 mb-4" />
     </template>
     <section v-else class="placeholder grey--text text--darken-3">
       <h4>Status Sidebar</h4>
       <v-icon>mdi-chevron-left</v-icon>
       <div class="info-content">{{ emptyMessage }}</div>
     </section>
+    <activity-discussion
+      :activity="selectedActivity"
+      panel
+      class="mt-2 mb-5 mx-1 py-2 px-4" />
```

Comment:
Should the padding be defined within the `Discussion` component since it affects layout of the component and not the surrounding layout?